src/app/lib/utilities/findNClosestIndicies/index.js

```diff
@@ -0,0 +1,25 @@
+// Given an array, radiate out bidirectionally from a starting index
+// Find the indices of the n closest elements that fulfill a predicate
+export default ({ array, startingIndex, n, predicate }) => {
+  const results = [];
+  if (predicate(array[startingIndex])) results.push(startingIndex);
+  let leftPointer = startingIndex - 1;
+  let rightPointer = startingIndex + 1;
+
+  while (leftPointer >= 0 || rightPointer < array.length) {
+    const searchLeft = leftPointer >= 0;
+    if (searchLeft > 0 && predicate(array[leftPointer])) {
```

Comment:
```suggestion
    if (searchLeft && predicate(array[leftPointer])) {
```
This confused me a bit but looks like artefact of development. So leftPointer is always boolean so this results in:
`true > 0` which in javascript is `true` luckily
OR
`false > 0` which in javascript is `false` luckily

So I think we can safely remove the `> 0` as it doesn't have any functional purpose